CHANGELOG.md

```diff
@@ -0,0 +1,162 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
```

Comment:
Do we really want this preface text at the top? Other notable projects like cargo don't have anything like that in [their changelog](https://github.com/rust-lang/cargo/blob/master/CHANGELOG.md).

If anything I think an adaptation of this section could live in the main readme. Perhaps something like the following at the bottom of the first introduction section:
```markdown
Druid adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html) and documents
all its changes in [the changelog](https://github.com/xi-editor/druid/blob/master/CHANGELOG.md).
```

Reply:
I've removed the preface.
Linking the changelog seams like a good idea.
Considering that semantic versioning is standard for Rust anyways I don't think that has to be mentioned.

Reply:
I've now also added a link to the readme.